.github/workflows/integration_test.yml

```diff
@@ -0,0 +1,53 @@
+name: Integration Test
+
+on: push
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
```suggestion
    name: Integration Test
    runs-on: ubuntu-latest
```

Reply:
naming for easier readability. Would be nice to have in the unit test also :)

Reply:
We thought about also having the unit test in the same workflow, it would overall not be a big problem but we just decided to separate them for clarity.

Reply:
thats fine, but adding the name tag to the job, will make it a bit easier to read, instead of it being named build :)

Reply:
I just mean, to add the name tag in the unit test file also